Getting Started: starlette (#7835)
getsentry/sentry-docs: src/platforms/python/guides/starlette/index.mdx

## Configure

> If you have the `starlette` package in your dependencies, the Starlette integration will be enabled automatically. There is nothing to do for you except initializing the Sentry SDK.

Reviewer: ```suggestion
If you have the `starlette` package in your dependencies, the Starlette integration will be enabled automatically when you initialize the Sentry SDK.
```

])
```

> When you point your browser to [http://localhost:8000/sentry-debug](http://localhost:8000/sentry-debug) a transaction in the Performance section of [sentry.io](https://sentry.io) will be created. Additionally an error event will be sent to [sentry.io](https://sentry.io) and will be connected to the transaction.

Reviewer: ```suggestion
When you point your browser to [http://localhost:8000/sentry-debug](http://localhost:8000/sentry-debug) a transaction will be created in the Performance section of [sentry.io](https://sentry.io). Additionally, an error event will be sent to [sentry.io](https://sentry.io) and will be connected to the transaction.
```
